extern/h5part/AUTHORS

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Is it a copy of https://github.com/quinoacomputing/H5Part ?
Can you use it as a submodule ?

Reply:
No. H5Part was originally a stand-alone library, but the developers added support for other types of data than particles (grids), and renamed it to H5Hut and it is maintained here https://gitlab.psi.ch/H5hut/src/-/wikis/home but it hardly changes much. Some years ago, I extracted the minimal files needed for a plugin build for paraview for my H5Part reader/writers and I added CMake build support. The files I have added to sph-exa are these minimal files needed and they are based on h5part 1.6.x. The quinoa library you linked to is the complete h5part 1.6.x distribution without cmake support in its original form (prior to it becoming H5Hut). Most of the material in that repo is the same, but would require a bit more work to integrate into sph-exa. In my opinion, easier to just use this until we need something like ADIOS.